Tone/instrument/Sampler.ts

```diff
@@ -357,6 +393,90 @@ export class Sampler extends Instrument<SamplerOptions> {
 		return this._buffers.loaded;
 	}
 
+    /**
+     * Set the loop start and end. Will only loop if loop is set to true.
+     * @param loopStart The loop start time
+     * @param loopEnd The loop end time
+     * @example
+     * const sampler = new Tone.Sampler("https://tonejs.github.io/audio/berklee/guitar_chord4.mp3").toDestination();
```

Comment:
nit: this `@example` isn't a valid use of the API. i think it'd need to look like: 

```ts
const sampler = new Tone.Sampler({
	urls: {
		A1: "https://tonejs.github.io/audio/berklee/guitar_chord4.mp3",
	},
}).toDestination();
// loop between the given points
sampler.setLoopPoints(0.2, 0.3);
sampler.loop = true;
```